_posts/cleaning-up-your-test-data-builders.md

```diff
@@ -0,0 +1,104 @@
+---
+layout: post
+title: "Cleaning Up Your Test Data Builders"
+draft: true
+tags: [ "Cleaner Code", "Test Automation" ]
+---
+
+I've been using the [Test Data Builder](http://www.natpryce.com/articles/000714.html) pattern for quite a while now. It's really improved the readability of my test automation code. I've always had the nagging feeling though that it could be better.
```

Comment:
How did it improve your test automation code? As a reader that is not using any test builders yet I might not see why you are using a builder.